### PR TITLE
Fix CSS typo

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Dumb typo. I had written:

```
--secondary-accent: var(--secondary-accent);
```

Instead of 

```
--secondary-accent: #d72451;
```

### CODE REVIEW

**Comment by jonbell-lot23** [comment]
It's late on the east coast, this is a regression, and it's a single line revert, so I am going to merge without review once tests pass.

File: packages/replay-next/pages/variables.css [hunk 1]
```diff
@@ -48,7 +48,7 @@
   --primary-accent: #01acfd;
   --primary-accent-hover: #0194ff;
 
-  --secondary-accent: var(--secondary-accent);
+  --secondary-accent: #d72451;
```
Comment (bvaughn) [inline packages/replay-next/pages/variables.css:51]: Hmm, this PR didn't get reviewed, and it seems to have set secondary accent and secondary accent hover to the same color.

Comment (bvaughn) [inline packages/replay-next/pages/variables.css:51]: Ignore my initial, stupid comment.

Anyway, after this PR both `--secondary-accent` and `--secondary-accent-hover` are the same color, which seems wrong.

Comment (jonbell-lot23) [reply packages/replay-next/pages/variables.css:51]: Will investigate

**Pull request merged**